qgrid/grid.py

```diff
@@ -1554,7 +1528,7 @@ def add_row(self, row=None):
         if row is None:
             added_index = self._add_empty_row()
         else:
-            added_index = self._add_empty_row()
```

Comment:
seem add_row method is not referred in any way, the current [add_row ](https://github.com/xellgrid/XellGrid/pull/9/files#diff-57f868d4357688ef72d5fa006b0c83f0e6063beca5dd420316ebb3335344dbe3R1398)content type is actually triggering duplicate_last_row which is confusing. we might need to redesign this communication.

Reply:
_add_empty_row is an experimental function I added some time back to test communication between backend and front end.  I will try to find a better way handle "add rows" operations.